CONTRIBUTING.rst

```diff
@@ -357,9 +357,9 @@ The tag format consists of a tag_prefix (<package name>-) and the release versio
   This Workflow first installs all dependencies, then allows for future versions of the dependencies and builds the package. After that the package is published to `pip`_.
 3. Automatically create conda-forge package
   This release than is recognized by a conda-forge bot, which triggers a new pull request for the conda-forge package and merges automatically if all tests pass.
-  These tests are defined as `GitHub-Action-Workflows`_ and are triggered for every new pull request. More information can be found in the next chapter.
+  Check if all tests as explained in the next chapter have passed
```

Comment:
Move this to the very beginning of the section, should be done before the tagging.

Reply:
The complete point 3 or just this sentence?

Reply:
I expect you want to have the complete point 3 as point 1. Am I right?

Reply:
I have now moved the single sentence to the top of the section.